project/Dependencies.scala

```diff
@@ -71,12 +73,14 @@ object Dependencies {
     r2dbcPool,
     r2dbcPostgres,
     akkaProjectionCore,
+    akkaProjectionGrpc,
     TestDeps.akkaProjectionEventSourced,
     TestDeps.akkaProjectionDurableState,
     TestDeps.akkaStreamTestkit,
     TestDeps.akkaTestkit,
     TestDeps.akkaProjectionTestKit,
     TestDeps.akkaJackson,
+    TestDeps.akkaDiscovery,
```

Comment:
why is this needed?

Reply:
Not sure, tests started failing with version mismatch between discovery and the rest of the Akka deps, but Test/dependencyTree doesn't show it 🤔 

Does Akka gRPC pull it in for client service discovery perhaps? But no-show in dep tree is a mystery to me.

Reply:
ok, I don't know, maybe something that resolves differently for milestones (I think I have seen such problems before with milestones)